src/index.ts

```diff
@@ -449,6 +452,122 @@ export class Datastore {
         await this._contract.removeGroupFromFile(fileId, groupId)
     }
 
+    /**
+     * Add a new label to the Datastore
+     * @param {string} name Label name
+     * @param {string} color Label color
+     */
+    async createLabel(name: string, color: string) {
+        await this._initialize()
+
+        if (name.length > 28)
+            throw 'Label name must not exceed 28 characters.'
+        
+        let hexColor = Color(color).hex()
+        await this._contract.createLabel(name, hexColor)
+    }
+
+    /**
+     * Delete a label from the Datastore
+     * @param {number} labelId Label Id
+     */
+    async deleteLabel(labelId: number) {
+        await this._initialize()
+
+        await this._contract.deleteLabel(labelId)
+    }
+
+    /**
+     * Assign a label to a file
+     * @param {number} fileId File Id
+     * @param {number} labelId Label Id
+     */
+    async assignLabel(fileId: number, labelId: number) {
+        await this._initialize()
+
+        await this._contract.assignLabel(fileId, labelId)
+    }
+
+    /**
+     * Unassign a label from a file
+     * @param {number} fileId File Id
+     * @param {number} labelIdPosition Label Id's position
+     */
+    async unassignLabel(fileId: number, labelId: number) {
+        await this._initialize()
+
+        let fileLabels = await this._contract.getFileLabelList(fileId)
+        let labelIdPosition
+        for (let i = 0; i < fileLabels.length; i++) {
+            if (labelId === fileLabels[i])
+                labelIdPosition = i
+        }
+        if (labelIdPosition)
+            await this._contract.unassignLabel(fileId, labelIdPosition)
+    }
+
+    /**
+     * Returns the wanted label
+     * @param labelId Label Id
+     */
+    async getLabel(labelId: number) {
+        await this._initialize()
+
+        const web3 = (window as any).web3
+        let label = await this._contract.getLabel(labelId)
+        let name = web3.toUtf8(label[0]);
+        let color = label[1].slice(2)
+        return {
+            labelId,
```

Comment:
What do you think if we return `id` instead of `labelId`?
```suggestion
            id: labelId,
```
It would reduce repetitions like `label.labelId` and would be more consistent with our other objects like `file` and `group`.